.github/workflows/ci.yml

```diff
@@ -151,7 +151,7 @@ jobs:
         pip-version:
           - latest
     env:
-      TOXENV: pip${{ matrix.pip-version }}
```

Comment:
```
874 passed, 1791 warnings in 997.59s (0:16:37)
```

Wow, [17 minutes](https://github.com/jazzband/pip-tools/actions/runs/5907461564/job/16025453351#step:8:127) on pypy with coverage vs [4 minutes](https://github.com/jazzband/pip-tools/actions/runs/5860759916/job/15889492766#step:8:104). 

@webknjaz, do you remember why the coverage wasn't set up in the first place? The speed?

Reply:
Since we don't have any `pypy` specific code, we can simply turn off the coverage there.

Reply:
I guess because of the slowness.

Reply:
Fixed in https://github.com/jazzband/pip-tools/pull/1970.